fixtures/preserve_comments-fixed_v2.yml

```diff
@@ -35,10 +35,7 @@ spec:
   containers:
   - name: test-container
     image: k8s.gcr.io/busybox
-    command:
-    - /bin/sh
-    - -c
-    - ls /etc/config/
+    command: ["/bin/sh", "-c", "ls /etc/config/"]
```

Comment:
These changes are kind of surprising, since the library is supposed to preserve the structure of the YAML file? Is this caused by the move to yaml.v3?

Reply:
Actually this output matches the formatting of the input file https://github.com/Shopify/kubeaudit/blob/master/fixtures/preserve_comments_v2.yml#L35
Our fork used default formatting for all data types so all arrays got marshaled into the multi-line dashed formatting. It wasn't great but it wasn't technically incorrect as far as yaml parsers could tell.

Reply:
I like the one liner way more :)